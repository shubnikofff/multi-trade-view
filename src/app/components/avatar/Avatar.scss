$dimension: 30px;

.avatar {
  border-radius: 50%;
  width: $dimension;
  height: $dimension;
}
