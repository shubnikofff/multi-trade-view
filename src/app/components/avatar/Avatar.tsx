import React from 'react';

import './Avatar.scss';

interface AvatarProps {
    url: string;
}

function Avatar({ url }: AvatarProps) {
    return (
        <img src={url} className="avatar" alt="Avatar" />
    );
}

export default Avatar;
