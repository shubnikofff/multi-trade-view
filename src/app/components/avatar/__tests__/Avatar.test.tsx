import React from 'react';

import { shallow } from 'enzyme';

import Avatar from '@components/avatar';

describe('Component Avatar', () => {
    const wrapper = shallow(<Avatar url="some-url" />);

    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
