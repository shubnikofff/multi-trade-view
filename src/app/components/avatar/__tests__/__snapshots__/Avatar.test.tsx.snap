// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Avatar should match snapshot 1`] = `
<img
  alt="Avatar"
  className="avatar"
  src="some-url"
/>
`;
